### ISSUE TITLE
Filter at Filebrowser

### ISSUE THREAD

**Issue by RockyJE** [gsantner/markor#1322]
Version v2.6.0 (122) F-Droid
Android 11

When opening a folder I would like to only see the .md files.
I keep my attachments (pics, pdfs, etc.) in the same folder as the .md files to which they are attached AND with the same file names (different extensions, obviously), and they clutter the file list.  

Perhaps I'm 'doing it all wrong', and should have a seperate folder for attachments; or am I missing something?

**Comment by RockyJE** [comment]
Thank you.  That is what I did at first, but was having trouble with the Markor File Manager recognizing a few of the .jpgs and .pngs I wanted to attach.  During troubleshooting, to make it quicker/eliminating taps, I moved those files to the same folder.  I guess I got carried away. 😀

I don't know why some pics weren't recognized, and it was weeks ago, so can't provide logs or anything.  In the future if I have troubles I'll generate a log.

Thank you for this wonderful app.  Coming from Joplin I find it to be refreshingly easy to use, i.e. no 'jibberish' filenames; it's easy to do a lot of stuff using my devices main file manager.
